### ISSUE TITLE
psycopg.org isn't up-to-date

### ISSUE THREAD

**Issue by devjev** [psycopg/psycopg2#1280]
Hi guys,

would it be possible to update the website with the newest documentation? The quick install section still shows incorrect code examples on how to use the _psycopg2-binary_ package:

```python
import psycopg

# Connect to your postgres DB
conn = psycopg.connect("dbname=test user=postgres")
...
```

Instead of

```python
import psycopg2

# Connect to your postgres DB
conn = psycopg.connect("dbname=test user=postgres")
...
```

But `install.rst` [here](https://github.com/psycopg/psycopg2/blob/master/doc/src/install.rst) is up-to-date.

**Comment by dvarrazzo** [comment]
Thank you for the report. Fixing it.

**Comment by dvarrazzo** [comment]
> But install.rst here is up-to-date.

I see: the typo was reported and fixed in b05a5819317ab19fc7749e34fb67f4604d9bd10a but I missed to cherry-pick it on the maint_2_8 branch, where the docs are published from.

**Comment by devjev** [comment]
Thank you!
